### PR TITLE
Update postgres-cdc demo can't with start incremental snapshot

### PR DESCRIPTION
**Pull request by syyfffy**
PostgresSourceBuilder's initing function is private . user can use like this to start  job with incremental snapshot

### CODE REVIEW

File: docs/content/connectors/postgres-cdc.md [hunk 1]
```diff
@@ -429,15 +429,15 @@ public class PostgresParallelSourceExample {
...
+                PostgresSourceBuilder.PostgresIncrementalSource.<String>builder()
                         .hostname("localhost")
                         .port(5432)
                         .database("postgres")
                         .schemaList("inventory")
                         .tableList("inventory.products")
                         .username("postgres")
                         .password("postgres")
-                        .slotName("flink")
+                        .slotName("flink2") // use a differnt slot.name for incremental snapshot
```
Comment (ruanhang1993) [inline docs/content/connectors/postgres-cdc.md:440]: use a different slot.name for each source to connect the same cluster

**Review by ruanhang1993 — COMMENTED** [review]
Thanks for the work. Left a little comment.

**Comment by syyfffy** [comment]
I change the comment and if u have free time ,please review it @ruanhang1993

**Review by GOODBOY008 — APPROVED** [review]
LGTM

**Review by gtk96 — APPROVED** [review]
thanks @syyfffy Thank you for repairing the mongo Chinese document.

**Comment by syyfffy** [comment]
@ruanhang1993  if u have free time, please review it and merge into master

**Pull request closed without merging**